.azure/scripts/shellcheck.sh

```diff
@@ -12,7 +12,9 @@ docker-images/kafka-based/kafka/stunnel-scripts/*.sh
 docker-images/kafka-based/kafka/exporter-scripts/*.sh
 docker-images/jmxtrans/*.sh
 tools/olm-bundle/*.sh
-tools/*.sh"
+tools/*.sh
+tools/cold-backup/run.sh
+tools/log-dump/run.sh"
```

Comment:
Should this instead of `run.sh` be `*.sh` to cover any scripts added int he future?

Reply:
OK, but what about any sub dir containing scripts?

Reply:
Well, I didnt said it would be perfect ... but it would be better. I'm not sure if it accepts something like `tools/log-dump/**/*.sh` or soemthing like thate.